cli/oci.go

```diff
@@ -251,10 +252,16 @@ func isCgroupMounted(cgroupPath string) bool {
 	if err := syscall.Statfs(cgroupPath, &statFs); err != nil {
 		return false
 	}
+	if goruntime.GOARCH == "s390x" {
```

Comment:
Again, this code should live in `cli/oci_s390.go`. Same comment for other `GOARCH` checks below.